zcash_client_sqlite/src/lib.rs

```diff
@@ -525,7 +523,12 @@ impl<C: Borrow<rusqlite::Connection>, P: consensus::Parameters> WalletRead for W
         account: AccountId,
         max_height: BlockHeight,
     ) -> Result<HashMap<TransparentAddress, NonNegativeAmount>, Self::Error> {
-        wallet::get_transparent_balances(self.conn.borrow(), &self.params, account, max_height)
+        wallet::transparent::get_transparent_address_balances(
```

Comment:
Why is this renaming the function in `wallet::transparent` so that it doesn't match the trait method? I think that for all of the other functions there that correspond to trait methods, the names do match.

Reply:
When moving the methods, I got confused about the meaning of `get_transparent_balances` vs `get_transparent_account_balance` and I wanted to clarify. I didn't want to change the top-level trait method, but since this is crate-private I thought that it was a good start.

Reply:
Do you want me to undo renaming it back?

Reply:
I don't feel strongly about it.